.flowconfig

```diff
@@ -3,10 +3,26 @@
 
 [ignore]
 ./dist/.*
+.*/assets/.*
 ./config/.*
 ./public/.*
 ./scripts/.*
-./node_modules/.*
+./ossl/.*
+./radata/.*
+.*/node_modules/.*
 
 [libs]
-./flow-typed/.*
\ No newline at end of file
+./flow-typed/.*
```

Comment:
I think we should remove this from the ignore. https://stackoverflow.com/questions/54327236/should-i-keep-flow-typed-npm-folder-versionated-in-git

Reply:
This is under `[libs]` so `flow` can lookup for types definitions inside `flow-typed` folder.

Also I added the folder to the repo, so we can version it. And it's not listed in the `.gitignore` file either.

Reply:
You're right. My bad